wasmcloud_host/lib/wasmcloud_host/lattice/state_monitor.ex

```diff
@@ -257,17 +257,18 @@ defmodule WasmcloudHost.Lattice.StateMonitor do
     actor_map =
       actors
       |> Enum.reduce(%{}, fn actor, actor_map ->
-        actor_id = Map.get(actor, "actor")
-        existing_actor = current_host |> Map.get(:actors, %{}) |> Map.get(actor_id)
+        actor_id = Map.get(actor, "public_key")
 
-        if existing_actor != nil && Map.get(existing_actor, :count) == Map.get(actor, "instances") do
-          Map.put(actor_map, actor_id, existing_actor)
-        else
-          Map.put(actor_map, actor_id, %{
-            count: Map.get(actor, "instances"),
-            status: "Awaiting"
-          })
-        end
+        actor_info = Map.get(actor_map, actor_id, %{})
+        count = Map.get(actor_info, :count, 0) + 1
```

Comment:
There's no `count` field on the list of actors that comes with the new heartbeat. The new heartbeat `actors` field is a list of `%{public_key: _, instance_id: _}` , as shown here https://github.com/wasmCloud/wasmcloud-otp/blob/main/host_core/lib/host_core/heartbeat_emitter.ex#L41

If you want the count of each unique actor on the host then you'll need to do a `Enum.group_by` on the actor's public key and count the instances (values).